src/queries/role.js

```diff
@@ -64,6 +66,13 @@ function queryAllByRole(
     }
   }
 
+  if (current !== undefined) {
```

Comment:
Is there any reason not to merge these two `if`s into one?

Reply:
Readability. That way we have a block for every option that is trivial to parse.